Rakefile

```diff
@@ -2,7 +2,27 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'solr_wrapper/rake_task' unless Rails.env.production?
+require 'rspec/core/rake_task'
 
 Rails.application.load_tasks
 
-require 'solr_wrapper/rake_task' unless Rails.env.production?
+Rake::Task["test"].clear
+
+desc 'Execute the test build that runs on travis'
```

Comment:
Or just 'run tests', since it should work anywhere

Reply:
you are right, i just got this from earthworks